src/Microsoft.DotNet.Build.Tasks/PackageFiles/partialfacades.targets

```diff
@@ -9,35 +9,6 @@
     <IgnoreArchitectureMismatches>true</IgnoreArchitectureMismatches>
   </PropertyGroup>
 
-  <!-- It seems this needs to be set here, and cannot be set in individual project files -->
-  <PropertyGroup Condition="'$(IgnoreArchitectureMismatches)' == 'true'">
-    <!-- Needed because we are using mscorlib.dll from the CoreCLR package, rather than a true reference assembly -->
-    <ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>None</ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>
-  </PropertyGroup>
-
-  <!-- PostFilterNuGetReferences
-       Filters the set of references generated from the ResolveNugetPackages target. This is a default target for partial facade
-         assemblies, and can be optionally used in other projects as well, by setting PostFilterNugetReferences=true. The target
-         contains some logic for removing well-known assemblies from the "Microsoft.DotNet.CoreCLR" package, except for "mscorlib.dll".
-  -->
-  <Target Name="PostFilterNugetReferences"
```

Comment:
While this isn't an issue for CoreCLR when we enabling this for .NET Native and Desktop I do think I want to figure out a good way to filter to only the references that are explicitly listed in the partial facade project. I don't want people to blindly have access to all the types in the full framework for example.